### ISSUE TITLE
Expose Internal Load Parameters?

### ISSUE THREAD

**Issue by chriswmackey** [chriswmackey/Dragonfly#19]
Right now, I don't expose these parameters, which are inputs in the .uwg file:

```
uwg.autosize = 0.
uwg.sensOcc = 100.
uwg.LatFOcc = 0.3
uwg.RadFOcc = 0.2
uwg.RadFEquip = 0.5
uwg.RadFLight = 0.7
```

I feel that people would prefer to have the ability to customize the actual internal loads and schedules before I expose these things but, if I am mistaken, please comment on this issue and let me know.

**Comment by saeranv** [comment]
@chriswmackey, I agree, the internal loads and schedules (by building type) are a lot more useful to modify then these parameters.
